Packs/CommonScripts/Scripts/CommandLineAnalysis/README.md

```diff
@@ -0,0 +1,67 @@
+This script evaluates command-line threats by analyzing both original and decoded inputs. It assigns weighted scores to detected patterns, such as AMSI bypass or credential dumping, and applies risk combination bonuses for multiple detections. The total score is normalized to a 0-100 scale, with risk levels categorized as follows:
+
+* 0-25: Low Risk
+* 26-50: Medium Risk
+* 51-90: High Risk
+* 91-100: Critical Risk
+
+The scoring mechanism provides a comprehensive risk assessment, considering both the severity and frequency of malicious behaviors.
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Cortex XSOAR Version | 6.10.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| command_line | The command line input to analyze. |
+| custom_patterns | A list of custom regex patterns to search for within the command line. Each pattern should be a valid regular expression string. |
+
+## Outputs
+
+---
+
+| **Path** | **Description** | **Type** |
+| --- | --- | --- |
+| CommandLineAnalysis.original_command | The original command line string analyzed for potential risks. | Unknown |
```

Comment:
One more thing that I noticed - all of the `Type` values in for context output this readme are "Unknown".
Is this intentional? I would assume most of the values would be "String", like so:
```suggestion
| CommandLineAnalysis.original_command | The original command line string analyzed for potential risks. | String |
```
The main options for `Type` in this case are usually: String, Number, Array or Boolean and Unknown.